lib/hitnmiss/file_driver.rb

```diff
@@ -0,0 +1,96 @@
+require 'fileutils'
+
+module Hitnmiss
+  class FileDriver
+    include Hitnmiss::Driver::Interface
+
+    def initialize(folder)
+      @folder = File.expand_path(folder)
+      FileUtils.mkdir_p(@folder)
+    end
+
+    def set(key, entity)
+      cache = { 'value' => entity.value }
+      if entity.expiration
+        expiration = Time.now.to_i + entity.expiration
```

Comment:
Figured since you were using it in the `internal_timestamp` probably would want to be consistent.